### ISSUE TITLE
Ophyd raises warning about signal_names when tab completing

### ISSUE THREAD

**Issue by apra93** [slaclab/HXRSnD#39]
## Expected Behavior

Typing in `motor.expert_screen()` should bring up the expert screen for the attocube motors.
Tab completion should simply display all the completions without any warnings.

## Current Behavior

Raises the following warning whenever tab completion is attempted:

```
/reg/g/pcds/pyps/conda/py36/envs/pcds-0.2.0/lib/python3.6/site-packages/ophyd/device.py:738: 
UserWarning: 'signal_names' has been renamed 'component_names' for clarity because it may 
include a mixture of Signals and Devices -- any Components. This alias may be removed in a 
future release of ophyd.
warnings.warn("'signal_names' has been renamed 'component_names' for "
```

## Possible Solution

Look for where in HXRSnD `signal_names` is still being used and replace it with `component_names`

**Comment by ZLLentz** [comment]
https://stackoverflow.com/questions/9031783/hide-all-warnings-in-ipython

**Comment by apra93** [comment]
Addressed using @ZLLentz fix in #47
